### PR TITLE
New page under demo for links of introduction and demo videos.

### PR DESCRIPTION
**Pull request by EthanChangAED**
Would like to suggest we include the links of several key introduction videos in the documentation.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ Akri's documentation is divided into six sections:
 
 1. 📘 [User Guide](./user-guide/getting-started.md): Documentation for Akri users.
 1. 🔎 [Discovery Handlers](./discovery-handlers/onvif.md): Documentation on how to configure Akri using Akri's currently supported Discovery Handlers
-1. 🚀 [Demos](./demos/usb-camera-demo.md): End-to-End demos that demostrate how Akri can discover and use devices. Contain sample brokers and end applications.
+1. 🚀 [Demos](./demos/video-links.md): Overall introduction videos and end-to-end demos that demostrate how Akri can discover and use devices. Contain sample brokers and end applications.
```
Comment (kate-goldenring) [inline docs/README.md:46]: I'd prefer to leave this as linking to the usb demo and keeping that first under the section as it is developer documentation

File: docs/demos/video-links.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Introduction and Demo Videos
+
+There are several on-line videos available to help you quickly understand the concept of Akri and get some quick demos.
+
+1. [Bridge Your IoT Leaf Devices to Local Clusters with Ease Using Akri and Dynamic Resource Allocation](https://www.youtube.com/watch?v=ZZfDne7gMMI) - Latest Akri introduction at KubeCon EU 2024.
+1. [Akri is dope!](https://www.youtube.com/watch?v=KLA3N8xbYPE) - A talk in the Azure Arc Jumpstart channel. Includes a demo of discovering an ONVIF camera with Akri and feeding the stream to a edge AI model.
```
Comment (kate-goldenring) [inline docs/demos/video-links.md:6]: To be honest, I don't love the title of this video. If we are going to link to it, can we not add it's title to the documentation?

File: docs/demos/video-links.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Introduction and Demo Videos
+
+There are several on-line videos available to help you quickly understand the concept of Akri and get some quick demos.
+
+1. [Bridge Your IoT Leaf Devices to Local Clusters with Ease Using Akri and Dynamic Resource Allocation](https://www.youtube.com/watch?v=ZZfDne7gMMI) - Latest Akri introduction at KubeCon EU 2024.
+1. [Akri is dope!](https://www.youtube.com/watch?v=KLA3N8xbYPE) - A talk in the Azure Arc Jumpstart channel. Includes a demo of discovering an ONVIF camera with Akri and feeding the stream to a edge AI model.
+1. [Connecting to devices at the edge with minimal footprint using AKS Edge Essentials and Akri on WASMs](https://www.youtube.com/watch?v=L6IhonIUOKU) - This talk goes over how users can use AKS Edge Essentials with WASM workloads to deploy a cluster on “light” edge hardware (PC-class devices) and run Akri, minimizing the footprint when connecting to these devices and utilizing them in your applications.
```
Comment (kate-goldenring) [inline docs/demos/video-links.md:7]: I wonder if we should avoid having 2 of the 3 videos be Azure focused. If we do that, we should also have a SUSE one.

File: docs/demos/video-links.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# Introduction and Demo Videos
+
+There are several on-line videos available to help you quickly understand the concept of Akri and get some quick demos.
```
Comment (kate-goldenring) [inline docs/demos/video-links.md:3]:
```suggestion
If you prefer to learn through videos rather than written documentation, the following is a list of informative talks and demos on Akri.
```

File: docs/SUMMARY.md [hunk 5]
```diff
@@ -18,6 +18,7 @@
 
 ## Demos
 
+* [Introduction and Demo Videos](demos/video-links.md)
```
Comment (kate-goldenring) [inline docs/SUMMARY.md:21]: I am not sure this is where this fits best, but i cannot come up with a good alternative. Can we move this to the bottom of this list as it is the odd ball out?

**Review by kate-goldenring — CHANGES_REQUESTED** [review]
Thank you for linking some videos. My main concern was the specific selection of videos. @yujinkim-msft as you are in all of them, I'd love to hear your thoughts

File: docs/demos/video-links.md (hunk 2 above)
Comment (yujinkim-msft) [reply docs/demos/video-links.md:6]: I'm aligned with this, let's just call it "Azure Arc Jumpstart with Akri" or something like that

File: docs/demos/video-links.md (hunk 3 above)
Comment (yujinkim-msft) [reply docs/demos/video-links.md:7]: can we add the newest kubecon talk with SUSE to the list of videos: https://youtu.be/ZZfDne7gMMI?si=DPwrY2VdwSP4-y4E

Comment (EthanChangAED) [reply docs/demos/video-links.md:7]: > can we add the newest kubecon talk with SUSE to the list of videos: https://youtu.be/ZZfDne7gMMI?si=DPwrY2VdwSP4-y4E

The current #1 is the kubecon talk.

Comment (EthanChangAED) [reply docs/demos/video-links.md:7]: > I wonder if we should avoid having 2 of the 3 videos be Azure focused. If we do that, we should also have a SUSE one.

I'll check with Nicolas for SUSE's application. If there is none, we can remove one of the Azure related link.

File: docs/README.md (hunk 1 above)
Comment (EthanChangAED) [reply docs/README.md:46]: Reversed this link and description.

**Comment by EthanChangAED** [comment]
> Thank you for linking some videos. My main concern was the specific selection of videos. @yujinkim-msft as you are in all of them, I'd love to hear your thoughts

While I'll check with SUSE for more videos if available. Is there any more we should put on the list? I was aware of the following one in 2022 that provides more details. If the contents are mostly up-to-date, we shall include it as well.
https://www.youtube.com/watch?v=9wCQCV0m5Kk

File: docs/README.md [hunk 6]
```diff
@@ -50,4 +50,4 @@ Akri's documentation is divided into six sections:
 
 ## Trademark
 
-The Linux Foundation has registered trademarks and uses trademarks. For a list of trademarks of The Linux Foundation, please see our [Trademark Usage page](https://www.linuxfoundation.org/legal/trademark-usage)
\ No newline at end of file
+The Linux Foundation has registered trademarks and uses trademarks. For a list of trademarks of The Linux Foundation, please see our [Trademark Usage page](https://www.linuxfoundation.org/legal/trademark-usage)
```
Comment (kate-goldenring) [inline docs/README.md:53]: @EthanChangAED are these changes needed here? Can they be reverted?

File: docs/demos/video-links.md (hunk 3 above)
Comment (kate-goldenring) [reply docs/demos/video-links.md:7]: There are older ones that provide a good overview of the project that Edrick, Jiri, or I have given that come up with a "Akri cncf" YouTube search. But i am not sure what our video selection criteria is here. Is it to put all the videos or one from each area of Akri?

File: docs/demos/video-links.md [hunk 7]
```diff
@@ -0,0 +1,9 @@
+# Introduction and Demo Videos
+
+If you prefer to learn through videos rather than written documentation, the following is a list of informative talks and demos on Akri.
+
+1. [Bridge Your IoT Leaf Devices to Local Clusters with Ease Using Akri and Dynamic Resource Allocation](https://www.youtube.com/watch?v=ZZfDne7gMMI) - Latest Akri introduction at KubeCon EU 2024.
+1. [Akri is dope!](https://www.youtube.com/watch?v=KLA3N8xbYPE) - A talk in the Azure Arc Jumpstart channel. Includes a demo of discovering an ONVIF camera with Akri and feeding the stream to a edge AI model.
+1. [Connecting to devices at the edge with minimal footprint using AKS Edge Essentials and Akri on WASMs](https://www.youtube.com/watch?v=L6IhonIUOKU) - This talk goes over how users can use AKS Edge Essentials with WASM workloads to deploy a cluster on “light” edge hardware (PC-class devices) and run Akri, minimizing the footprint when connecting to these devices and utilizing them in your applications.
+
+To try more demos/examples with step-by-step guidance, check the rest of the pages under **Demo** section.
```
Comment (kate-goldenring) [inline docs/demos/video-links.md:9]: Can you add a link for "Demo section"?

**Review by kate-goldenring — APPROVED** [review]
